src/mixins/createFormFieldMixin.ts

```diff
@@ -76,52 +102,38 @@ const createFormMixin: FormMixinFactory = compose({
 					this.setState({ value: stringToValue(event.value) });
 				}
 			}
-		}
-	}, (instance: FormField<any>, options: FormFieldMixinOptions<any, FormFieldMixinState<any>>) => {
-		if (options) {
-			const { type } = options;
-			if (type) {
-				instance.type = type;
+		},
+
+		nodeAttributes: [ function (this: FormFieldMixin<any, FormFieldMixinState<any>>): VNodeProperties {
```

Comment:
Hmmm... good point.  I suspect we always haven't been clear.

To try to state it, if the items in the literal can be easily expressed without line breaks, the literal can be expressed without a starting and terminating line break, though with potentially line breaks between the elements.  If all of the elements cannot be expressed without line breaks within the elements, then each element should start and finish with line breaks, including the leading and terminating line break for the literal.

TL;DR I agree with you and will change it.